lib/mappers/__tests__/__fixtures__/mapper.figma.js

```diff
@@ -0,0 +1,17 @@
+const sampleOutput = require('../../../../__mocks__/figma/sample_dump.json');
+const { figma2jsonFromJSON, findIds } = require('../../../figma');
+const { uniqueArray } = require('../../../utils');
+
+module.exports = () => {
+  figma2jsonFromJSON(sampleOutput);
```

Comment:
this is weird? What does this function do when you call it without storing its return value? None of the values below rely on what is returned from this function?